src/makeInfinityAdder.js

```diff
@@ -3,8 +3,14 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(sum = 0) {
```

Comment:
The initial value of `sum` should not be set to 0 by default in the `makeInfinityAdder` function. According to the task description, the adder function should not have a default value for its argument.